### DISCUSSION TITLE
Any plan to support typescript?

### DISCUSSION THREAD

**Discussion by dealdot** [colbyfayock/next-wordpress-starter#632] [Ideas]
Hello, does this project have any plan for supporting typescript, thanks

**Comment by colbyfayock** [comment]
hey @dealdot i haev no plans on adding typescript support for the current version

however i would have plans to add it to the App Router version that i started working on, but need to come back to

it's been low in my priorities lately so haven't been able to spend time on it for a while, but will eventually get back to it

in the meantime, id happily accept a PR converting the existing Pages router version to Typescript, which will still live long in a branch for people to use
